devices/power.go

```
@@ -20,8 +19,17 @@ var AmtPower = sdk.DeviceHandler{
 // amtPowerRead gets the current power state of the AMT device
 func amtPowerRead(device *sdk.Device) ([]*sdk.Reading, error) {
 
+	cmd := exec.Command("python", "scripts/power.py", device.Data["ip"], // nolint: gas
+		"Vap0rIsDataDriven!", "status")
+
+	out, err := cmd.Output()
+
+	if err != nil {
+		fmt.Printf("Error: %s\n", cmd.Stderr)
```

if we get an error here, we'll want to return the error, not just print it out.

we should avoid using `fmt.Printf` in the plugins, instead use the sdk logger